config/config/sc-config.yaml

```diff
@@ -597,3 +597,16 @@ metricsServer:
 
 calicoAccountant:
   enabled: true
+
+s3Exporter:
```

Comment:
Should we add image tag here?

Reply:
Hmm, not sure about this. We don't seem to expose tags for anything else?

Reply:
Right, but then I think that we should have a tag set for the chart values file. It doesn't feel very reassuring to always use latest

Reply:
It uses the charts app version by default so not the latest. :slightly_smiling_face: 
```
image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Reply:
Ooh, I missed that. Yea, that's fine then